template/src/App.tsx

```diff
@@ -20,14 +23,38 @@ interface State {
   // client: ApolloClient<InMemoryCache>;
 }
 
+// TODO(ratkinson): convert to FC
 class App extends Component<{}, State> {
```

Comment:
As a followup I intend to convert this to a functional component

Reply:
That will be a great update!